backends/cadence/fusion_g3/operators/op_quantize.cpp

```diff
@@ -802,6 +802,6 @@ Tensor& quantize_per_token_out(
 }
 
 } // namespace native
-} // namespace FusionG3
+} // namespace G3
 } // namespace impl
 } // namespace cadence
```

Comment:
Nit: missing new line at end of file

Reply:
> Nit: missing new line at end of file

All the operator files do not have a new line at the end. Not having a new line at the end is not shown as linter error.